modules/tinymce/src/core/test/ts/browser/fmt/MediaAlignTest.ts

```diff
@@ -0,0 +1,77 @@
+import { ApproxStructure } from '@ephox/agar';
+import { context, describe, it } from '@ephox/bedrock-client';
+import { Arr } from '@ephox/katamari';
+import { TinyAssertions, TinyHooks, TinySelections } from '@ephox/mcagar';
+
+import Editor from 'tinymce/core/api/Editor';
+import Theme from 'tinymce/themes/silver/Theme';
+
+type Alignment = 'left' | 'center' | 'right' | 'justify';
+
+describe('browser.tinymce.core.delete.MediaAlignTest', () => {
+  const hook = TinyHooks.bddSetupLight<Editor>({
+    base_url: '/project/tinymce/js/tinymce'
+  }, [ Theme ], true);
+
+  const mediaApproxStructure = (tag: string, alignment: Alignment) => {
+    const alignStyles = (str: ApproxStructure.StringApi) => ({
```

Comment:
if `alignment` is already in scope here why not make this function just return the style assertion directly? Replace this delayed object indexing with a switch statement and some `return`s

Reply:
Done